Dockerfile

```diff
@@ -17,12 +17,14 @@ RUN export GOOS=$TARGETOS && \
     export GOARM=$(echo ${TARGETPLATFORM} | cut -d / -f3 | tr -d 'v') && \
     make build
 
-FROM k8s.gcr.io/build-image/debian-iptables:bullseye-v1.1.0 AS nmi
+FROM k8s.gcr.io/build-image/debian-iptables:bullseye-v1.2.0 AS nmi
 # upgrading libssl1.1 due to CVE-2021-3711 and CVE-2021-3712
 # upgrading libgssapi-krb5-2 and libk5crypto3 due to CVE-2021-37750
 # upgrading libgmp10 due to CVE-2021-43618
 # upgrading bsdutils due to CVE-2021-3995 and CVE-2021-3996
-RUN clean-install ca-certificates libssl1.1 libgssapi-krb5-2 libk5crypto3 libgmp10 bsdutils
+# upgrading libc6 due to CVE-2022-23218
+# upgrading libc-bin due to CVE-2022-23218
```

Comment:
There are more than 1 CVE according to https://dev.azure.com/AzureContainerUpstream/AAD%20Pod%20Identity/_build/results?buildId=40850&view=logs&jobId=7e664a44-1d98-5348-601c-dff379a14ab5&j=7e664a44-1d98-5348-601c-dff379a14ab5&t=df0bac15-41b8-536a-5d4e-7c9def102aed:

```suggestion
# upgrading libc-bin and libc6 due to CVE-2021-33574, CVE-2022-23218, CVE-2022-23219 and CVE-2021-43396
```